src/renderer/atlas/AtlasEngine.cpp

```diff
@@ -631,13 +631,12 @@ void AtlasEngine::_flushBufferLine()
 

```

Comment:
A few more unit tests might help cover edge cases.

Reply:
I agree with you, however in this particular instance you'd have to tell me what the edge case is that causes the issue, because no one knows right now and it's not possible to write a test for something unknown.